sources/packages/web/src/views/student/StudentAccountActivity.vue

```diff
@@ -10,42 +10,26 @@
         :routeLocation="{ name: StudentRoutesConst.STUDENT_DASHBOARD }"
       />
     </template>
-    <tab-container :enableCardView="false" class="mb-6">
-      <student-scholastic-standing-limited-history
-    /></tab-container>
-    <formio-container
+    <student-account-restrictions />
+    <student-scholastic-standing-limited-history /><formio-container
```

Comment:
Please move `<formio-container formName="studentAccountActivity" />` to next line.